src/yamux/yamux.lua

```diff
@@ -411,6 +516,40 @@ local function parse_yamux(subtree, pinfo, fields, yamux_header)
     -- Update the last frame number that was processed by this dissector
     yamux_stream_info[current_stream_id].last_frame = current_frame
 
+    if not (disable_window_calc_logic or is_yamux_stream_default_created()) then
+        -- Window values recalculation
+        local new_server_window_size = window_start_value
+        local new_client_window_size = window_start_value
+
+        if previous_frame then
+            -- Use prev values, if prev frame is presented
+            new_server_window_size = yamux_stream_info[current_stream_id].server_window_size
+            new_client_window_size = yamux_stream_info[current_stream_id].client_window_size
+        end
```

Comment:
```suggestion
        if previous_frame and (yamux_stream_info[current_stream_id].client_window_size and yamux_stream_info[current_stream_id].server_window_size) then
            -- Use prev values, if prev frame is presented
            new_server_window_size = yamux_stream_info[current_stream_id].server_window_size
            new_client_window_size = yamux_stream_info[current_stream_id].client_window_size
        end
```
This assumes that the `client_window_size` and `server_window_size` variables have already been initialized for the stream.

When I ran the tests in this repo, I saw that it is possible with a partial packet capture that the dissector is able to correlate the frame to an existing stream that had not yet had the window size variables initialized. I suggest extending the conditional to ensure that the variables are not nil before trying to set them as the new window size.